tests/playwright/config/global-setup.js

```diff
@@ -2,14 +2,15 @@
 const { chromium } = require( '@playwright/test' );
 
 module.exports = async ( config ) => {
-	const browser = await chromium.launch();
-	const page = await browser.newPage();
-	await page.goto( `${ config.projects[ 0 ].use.baseURL }/wp-admin` );
+	const configCurrentUse = config.projects[ 0 ].use,
+		browser = await chromium.launch( { headless: configCurrentUse.headless } ),
+		page = await browser.newPage();
+
+	await page.goto( `${ configCurrentUse.baseURL }/wp-admin` );
 
 	await page.waitForSelector( 'text=Log In' );
-	await page.fill( 'input[name="log"]', process.env.UNAME || 'admin' );
-	await page.waitForTimeout( 500 );
```

Comment:
so the timeout is not necessary anymore? How did does it work now?

Reply:
I don't remember what fixed it, but it's not necessary and issue never occurs.